drivers/include/abp2.h

```diff
@@ -0,0 +1,313 @@
+/**
```

Comment:
GitHub Actions / static-tests

no copyright notice found

Reply:
Fix that one, and most other files, where the copyright was wrong.